app/services/unsubscribe_brf_service.rb

```diff
@@ -0,0 +1,79 @@
+class UnsubscribeBRFService
+  MESSAGE = <<~BODY.freeze
+    goes here!
+  BODY
+
+  SUBJECT = "subject goes here".freeze
+
+  def self.call
+    UnsubscribeBRFService.new.call
+  end
+
+  def call
+    subscriber_lists = SubscriberList.where("title like ?", "%Brexit guidance for your business%")
+    puts "Unsubscribing #{subscriber_lists.count} subscriptions"
+    subscriber_lists.inject(false) do |result, subscriber_list|
```

Comment:
This is cool,
So do i understand this correnctly `.inject(false)` takes the enumberable `subscriber_lists` and sets a default value of false for it... then overwrites that only if the block returns true?

Reply:
it's a way of or-ing all the results so we know if a message has been sent at all